### ISSUE TITLE
predict_proba for multilabel output?

### ISSUE THREAD

**Issue by bluetyson** [stevenpawley/Pyspatialml#25]
Hi Steven,

I was just wondering about this today ... what would you do, stick a check in and if there is a list there, extract all the positive probabilities for each output and shape into the multiclass proba shape - as for making a raster output, sort of the same thing?

**Comment by bluetyson** [comment]
Something like this? 


           ` if isinstance(result, list):
                result = np.array(result)[:,:,1].T`

**Comment by stevenpawley** [comment]
I'm not sure if I completely follow, but adding predict_proba for the multilabel case is something that I'm happy to look into and implement. I haven't had a need for it until now, but I'm currently working on something that could deal with multilabels.

**Comment by stevenpawley** [comment]
But, yes, something like the `isinstance(result, list)` to catch the output for multilabel cases

**Comment by RichardScottOZ** [comment]
Yes, that is what I meant Steven.  A hack version seemed to work as a test, anyway.
